common/OpTestSystem.py

```diff
@@ -1002,6 +1002,8 @@ def sys_ipmi_boot_system_to_petitboot(self, i_console):
         else:
             l_msg = "System failed to reach standby/Soft-off state"
             raise OpTestError(l_msg)
+
+        self.cv_IPMI.ipmi_set_boot_to_petitboot()
```

Comment:
Yep - any tagged Petitboot >= v1.0.1 has the fix, https://github.com/open-power/petitboot/commit/8569abdf356aa617ebdcb426aa708a0fd8129a1a